src/modules/internet/char-mappings.ts

```diff
@@ -0,0 +1,285 @@
+//selectively sourced from https://github.com/sindresorhus/transliterate/blob/main/replacements.js (under MIT license)
```

Comment:
```suggestion
// selectively sourced from https://github.com/sindresorhus/transliterate/blob/main/replacements.js (under MIT license)
```
Please add a space after the `//` on every comment. Otherwise, this LGTM